src/storage/examples/src/lib.rs

```diff
@@ -516,6 +516,9 @@ pub async fn run_object_examples(buckets: &mut Vec<String>) -> anyhow::Result<()
         .await?;
     tracing::info!("create test object for object ACL examples");
     let _ = make_object(&client, &id, "object-to-update").await;
+    let _ = make_object_with_acl(&client, &id, "object-with-acl").await;
+    tracing::info!("running print_file_acl example");
```

Comment:
We don't need any of this fancy set up. It is nice to see an ACL being printed, but not necessary for our tests.

I think we should put this next to `objects::get_metadata::sample(...)` and reuse "object-to-read" as the `NAME`.